src/genesys/webChat/client/src/genesysServiceDesk.ts

```diff
@@ -104,14 +98,27 @@ class GenesysServiceDesk implements ServiceDesk {
     this.callback = parameters.callback;
     this.initialAgentJoined = false;
   }
+
+  /**
+   * Returns a name for this service desk integration. This value should reflect the name of the service desk that is
+   * being integrated to. This information will be reported to IBM and may be used to gauge interest in various
+   * service desks for the possibility of creating fully supported out-of-the-box implementations.
+   *
+   * This value is required for custom service desks and may have a maximum of 40 characters.
+   */
+  getName() {
+    return 'genesys';
```

Comment:
Should we post-fix this with the TYPE on genesys sdk this is using?

Reply:
This comment goes for all of them, really. Is just having the SD name enough or do we want to be able to understand SDKs being used in the long run.

Reply:
Some of these only have one so I don't know what else we would call it. I tweaked a few of the names.